cdap-hbase-compat-0.94/src/main/java/co/cask/cdap/data2/util/hbase/HTable94NameConverter.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.data2.util.hbase;
+
+import co.cask.cdap.common.conf.Constants;
+
+/**
+ * Utility methods for dealing with HBase table name conversions in HBase 0.94.
+ */
+public class HTable94NameConverter extends HTableNameConverter {
+  public String getSysConfigTablePrefix(String tableName) {
+    return HBaseTableUtil.HBASE_NAMESPACE_PREFIX + Constants.SYSTEM_NAMESPACE + ".";
```

Comment:
So this is hard-coded to always use "cdap_".  Will the change to base that on the prefix from tableName come in a later PR?